### PR TITLE
build: rename ripple-lib to xrpl.js

### PR DESCRIPTION
**Pull request by natenichols**
## High Level Overview of Change
Renames `ripple-lib` to `xrpl.js` where neccesary.

### Context of Change
Renames `ripple-lib` to `xrpl` in package.json. 

Renames webpacked `ripple-latest.js` to `xrpl-latest.js`

Renames `ripple-lib` to `xrpl.js` in the documentation (README, APPLICATIONS)

### Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation Updates
- [ ] Release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,12 +79,12 @@ If you want to use `ripple-lib` with React Native you will need to have some of
...
 
-### Using ripple-lib with Deno
+### Using xrpl.js with Deno
 
-Until official support for [Deno](https://deno.land) is added, you can use the following work-around to use `ripple-lib` with Deno:
+Until official support for [Deno](https://deno.land) is added, you can use the following work-around to use `xrpl.js` with Deno:
 
 ```javascript
-import ripple from 'https://dev.jspm.io/npm:ripple-lib';
+import ripple from 'https://dev.jspm.io/npm:xrpl.js';
```
Comment (JST5000) [inline README.md:87]: Should this be `import xrpl` instead of `import ripple`?

**Review by JST5000 — COMMENTED** [review]
Not sure if the question I posed is a bug or feature :P - Rest looks good though!

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# ripple-lib (RippleAPI)
+# xrpl.js (RippleAPI)
```
Comment (mvadari) [inline README.md:1]:
```suggestion
# xrpl.js
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
 
 A JavaScript/TypeScript API for interacting with the XRP Ledger
```
Comment (mvadari) [inline README.md:3]:
```suggestion
A JavaScript/TypeScript library for interacting with the XRP Ledger
```

File: README.md [hunk 4]
```diff
@@ -79,12 +79,12 @@ If you want to use `ripple-lib` with React Native you will need to have some of
...
 
-Until official support for [Deno](https://deno.land) is added, you can use the following work-around to use `ripple-lib` with Deno:
+Until official support for [Deno](https://deno.land) is added, you can use the following work-around to use `xrpl.js` with Deno:
 
 ```javascript
-import ripple from 'https://dev.jspm.io/npm:ripple-lib';
+import ripple from 'https://dev.jspm.io/npm:xrpl.js';
 
 (async () => {
   const api = new (ripple as any).RippleAPI({ server: 'wss://s.altnet.rippletest.net:51233' });
```
Comment (mvadari) [inline README.md:90]: I assume we're not worried about changing sample code for now.

**Review by mvadari — APPROVED** [review]
A few minor comments, but nothing to hold up approval

File: README.md (hunk 4 above)
Comment (natenichols) [reply README.md:90]: I was not concerned about it... I also didn't change URLs or anything like that.

File: README.md [hunk 5]
```diff
@@ -31,22 +31,22 @@ What is ripple-lib used for? The applications on the list linked above use `ripp
 
 See also: [RippleAPI Beginners Guide](https://xrpl.org/get-started-with-rippleapi-for-javascript.html)
 
-In an existing project (with `package.json`), install `ripple-lib`:
+In an existing project (with `package.json`), install `xrpl.js`:
 ```
-$ npm install ripple-lib
+$ npm install xrpl
 ```
```
Comment (JST5000) [inline README.md:37]: These two seem out of sync -> is it install `xrpl.js` or install `xrpl`?

Comment (mvadari) [reply README.md:37]: it's `install xrpl`, but the package is named `xrpl.js` - the `.js` doesn't actually mean anything from a node point of view. 
see: https://web3js.readthedocs.io/en/v1.5.2/getting-started.html

**Review by JST5000 — APPROVED** [review]
LGTM!

**Pull request merged**